src/compute.py

```diff
@@ -15,6 +15,82 @@
 from hoomd.error import DataAccessError
 
 
+class VelocityCompute(hoomd.operation.Compute):
+    r"""Center-of-mass velocity compute.
+
+    Args:
+        filter (hoomd.filter.ParticleFilter): HOOMD particles to include.
+            The default value of `None` means no HOOMD particles are included.
+        include_mpcd_particles (bool): If `True`, include MPCD particles in
```

Comment:
Do you want to include default value?

Reply:
Since the default values of arguments get included in the function signatures, I've tended to not document these values in the docstrings (so I don't have to remember to update them if we change the default in future). Any defaults not in a function signature should continue to be documented explicitly, though.